### DISCUSSION TITLE
Issue with Bounds in SkBee

### DISCUSSION THREAD

**Discussion by Z-1-n** [ShaneBeee/SkBee#631] [Q&A]
I'm having trouble with the bound enter event; it just isn't called, at all. I get this error when loading SkBee, which I think is related.
![image_2024-05-12_184516394](https://github.com/ShaneBeee/SkBee/assets/148121776/5c6e020e-1f4a-412f-800f-ed77c590631c)

My bad if this is something I'm making a mistake with.

Here's Some info on my server version, Skript version & SkBee Version:
![image_2024-05-12_184554717](https://github.com/ShaneBeee/SkBee/assets/148121776/ff7b02b4-32ae-424c-866e-ac03edaefaec)
![image](https://github.com/ShaneBeee/SkBee/assets/148121776/0c48eb85-f723-4422-a692-10dd5abff159)
![image](https://github.com/ShaneBeee/SkBee/assets/148121776/fd2a6105-b94b-48f2-aee8-0e69f112e2a8)

**Comment by ShaneBeee** [comment]
please use the issues tab for reporting issues
